modules/ROOT/pages/osv2-apis.adoc

```diff
@@ -474,6 +479,53 @@ This request returns a response similar to the following:
 ]
 ----
 
+==== Example: Retrieve Usage Metrics by Region
+
+To view Object Store usage statistics for a region,
+submit a `curl` command similar to the following, 
+replacing ORG_ID with your organization ID, ENV_ID with your environment ID,
+and BEARER_TOKEN with your bearer token:
+
+[source,console,linenums]
+----
+curl -X GET \
+ 
+----
+
+This request returns a response similar to the following:
+
+[source,json,linenums]
+----
+[
+
+]
+----
+
+
+
+For the list of CloudHub deployment regions, see xref:runtime-manager::cloudhub-networking-guide.adoc#regional-services[Regional Services].
+
+==== Example: Retrieve Usage Metrics by Store
+
+To view Object Store usage statistics for a store,
+submit a `curl` command similar to the following, 
+replacing ORG_ID with your organization ID, ENV_ID with your environment ID,
```

Comment:
Add here same as before for region  id and also adding store id which to get metrics from